client/src/app/site/motions/services/motion-csv-export.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@angular/core';
+
+import { TranslateService } from '@ngx-translate/core';
+
+import { CsvExportService } from 'app/core/services/csv-export.service';
+import { ViewMotion } from '../models/view-motion';
+
+/**
+ *
+ */
+@Injectable({
+    providedIn: 'root'
+})
+export class MotionCsvExportService {
```

Comment:
since we potentially gonna have a lot of export-services we might create a base-csv-export class or an interface to predefined methods that should exist

Reply:
The problem is, that except `list` every export is speacial and currently I think an interface with just one method is pointless, if every has multiple special cases. We might think of this if more exports are available to see, how big the union of all classes are.